<launch>
	<!-- declare arguments with default values -->
	<arg name="fps" default="30"/>                  			<!--frame per second to try to acquire the image -->
	<arg name="CalibrationFile" default=""/>        			<!--Path to Autoware calibration format file-->
	<arg name="format" default="rgb8"/>              			<!--Pixel Format to acquire the image "raw" or "rgb"-->
	<arg name="timeout" default="1000"/>            		  	<!--Timeout in mili seconds-->
	<arg name="camera_selected" default="0"/>

	<node pkg="flir" type="bfly_node" name="bfly_node" output="screen">
		<param name="fps" value="$(arg fps)"/>
		<param name="calibrationfile" value="$(arg CalibrationFile)"/>
		<param name="format" value="$(arg format)"/>
		<param name="timeout" value="$(arg timeout)"/>
		<param name="camera_selected" value="$(arg camera_selected)"/>
	</node>
</launch>
